src/Cli/dotnet/commands/dotnet-fsi/FsiCommandParser.cs

```diff
@@ -2,14 +2,28 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using Microsoft.DotNet.Tools.Fsi;
 
 namespace Microsoft.DotNet.Cli
 {
     internal static class FsiCommandParser
     {
+        public static readonly string DocsLink = "https://aka.ms/dotnet-fsi";
```

Comment:
This link doesn't appear to be valid.

Reply:
@baronfel mind creating this link (or a new one if this one is already in use)?

Reply:
I went ahead and created it and targeted https://docs.microsoft.com/dotnet/fsharp/tools/fsharp-interactive/, @marcpopMSFT @baronfel you're owners so you should be able to change it if there's a more appropriate doc to link to.

Reply:
sorry, I missed this ping. That is the correct link, though!